### DISCUSSION TITLE
direkt serial rs232

### DISCUSSION THREAD

**Discussion by Mk2AGU** [davidrapan/ha-solarman#218] [Q&A] [answered]
Hello everyone, has anyone ever tried to connect the Deye directly via RS232/serial without using the Wi-Fi dongle? I have set this up in IPS (Symcon), but since I switched to Home Assistant, I'm looking for a solution to connect without the Wi-Fi dongle.

mfg

**Comment by davidrapan** [comment, marked as answer]
Hi, scope of this integration is only within solarman communication protocol.
